manifests/init.pp

```diff
@@ -80,10 +80,15 @@
           $_tools_x_package_name_default = 'open-vm-tools-desktop'
           $_service_name_default         = 'vmtoolsd'
         }
-        'SLED', 'SLES', 'OpenSuSE': {
+        'SLED', 'SLES': {
           $_tools_x_package_name_default = 'open-vm-tools-desktop'
           $_service_name_default         = 'vmtoolsd'
         }
+        'OpenSuSE': {
+          $_tools_x_package_name_default = 'open-vm-tools-gui'
+          $_service_name_default         = 'vmtoolsd'
+        }
+ 
```

Comment:
From travis:
manifests/init.pp - ERROR: two-space soft tabs not used on line 91
manifests/init.pp - ERROR: trailing whitespace found on line 91